test/e2e/config/config.go

```diff
@@ -144,6 +156,12 @@ func (c *ProviderTestingConfig) String() string {
 	return string(prettyConfig)
 }
 
+func (c *ClusterTestingConfig) SetDefaultArchitecture() {
+	if c.Architecture == "" {
+		c.Architecture = ArchitectureAmd64
```

Comment:
For the hosted case, wouldn't it be better to inherit "parent"'s architecture instead if nothing is set?